pubspec.yaml

```diff
@@ -9,8 +9,8 @@ authors:
   - Max Peterson <maxwell.peterson@workiva.com>
   - Trent Grover <trent.grover@workiva.com>
 homepage: https://github.com/Workiva/fluri
-dev_dependencies:
-  dependency_validator: ^1.1.2
-  test: '>=0.12.23+1 <2.0.0'
 environment:
-  sdk: ">=1.23.0 <3.0.0"
+  sdk: ">=2.2.0 <3.0.0"
```

Comment:
Is there a reason you've chosen `2.2.0` for the minimum?

Reply:
I was going to do 2.0.0 but the latest version of dep validator requires 2.2.0 so I figured that was a wide enough range 🤷‍♂

Reply:
I could also widen the dependency_validator range and move this back down to 2.0.0

Reply:
Cool cool cool. Should we suggest 2.2.0 as the floor in our cleanup doc?

Reply:
I think internally we should go straight to 2.4.1 since we're already using that everywhere

Reply:
In the future I think we should aim for auto-raising the SDK minimum when we propagate new versions of the SDK anyway, just to ensure that we're all using the same version.

Reply:
FWIW, I don't think it's worth going back and changing the SDK minimum in dependency_validator.